cdm/src/main/java/ucar/nc2/ft2/coverage/simpgeometry/SimpleGeometry.java

```diff
@@ -0,0 +1,20 @@
+package ucar.nc2.ft2.coverage.simpgeometry;
+
+import ucar.ma2.Array;
+
+/**
+ * An interface to interact with
+ * CF 1.8 Compliant Simple Geometries,
+ * and an Enum for the different
+ * geometries
+ *
+ * @author Katie
+ *
+ */
+enum CFGEOMETRY {

```

Comment:
It is usually easier to find classes, interfaces, enums, etc if they are located in dedicated files. Is there a strong argument for putting this enum in the same file as another interface?

Reply:
fixed.